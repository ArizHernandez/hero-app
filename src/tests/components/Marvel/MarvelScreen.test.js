const { shallow } = require("enzyme")
const { MarvelScreen } = require("../../../components/Marvel/MarvelScreen")

describe('Pruebas en el <MarvelScreen />', () => {
  
  const wrapper = shallow(<MarvelScreen />)

  test('Debe de cargar correctamente', () => {
    expect(wrapper).toMatchSnapshot();
  });

  test('Debe de cargar <HerosList />', () => {
    expect(wrapper.find('HerosList').exists()).toBe(true);
  });
})
