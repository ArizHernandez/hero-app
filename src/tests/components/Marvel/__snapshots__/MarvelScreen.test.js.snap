// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el <MarvelScreen /> Debe de cargar correctamente 1`] = `
<div
  className="container mt-3"
>
  <h1
    className="text-center"
  >
    Marve Screen
  </h1>
  <hr
    className="mt-0 mb-4"
  />
  <HerosList
    publisher="Marvel Comics"
  />
</div>
`;
